### DISCUSSION TITLE
Help with formatISO

### DISCUSSION THREAD

**Discussion by laurensiusadi** [js-fns/date-fns#2366] [Q&A] [answered]
I'm trying to use `formatISO` from [the docs](https://date-fns.org/v2.19.0/docs/formatISO)
The example as follows:
```
const result = formatISO(new Date(2019, 8, 18, 19, 0, 52))
//=> '2019-09-18T19:00:52Z'
```

But instead I'm getting `2019-09-18T19:00:52+07:00`

How do I get the format like in the docs?

**Comment by devuxer** [comment, marked as answer]
1. Easiest way I know of is to just use the built-in JS function`toISOString()`, though it's not *exactly* the same as the date-fns docs.

```ts
const result = new Date(2019, 8, 18, 19, 0, 52).toISOString();
// "2019-09-18T19:00:52.000Z"
```

2. You could kind of fudge it like this:

```ts
const result = format(new Date(2019, 8, 18, 19, 0, 52), "yyyy-MM-dd'T'HH:mm:ss'z'")
// "2019-09-18T19:00:52z"
```

3. Finally, you could go into your OS settings and change the time zone to "(UTC) Coordinate Universal Time" (or equivalent), then try your code again. You should get exactly what's in the date-fns docs. (This is probably not practical unless you can count on your code always running on a server with this setting.)

**Reply by safareli** [reply]
<img width="334" alt="Screen Shot 2021-06-16 at 10 24 47 PM" src="https://user-images.githubusercontent.com/1932383/122272502-ae33c280-cef1-11eb-8a1d-1f63f3c78012.png">

**Reply by laurensiusadi** [reply]
@safareli This will depends on your local UTC time. So the native Date method won't work.
I end up using the `formatISO` above with +00:00 timezone because it just works when you do `parseISO` it will transform back to correct Date object.

I found a way to get the result, but didn't use it. So I haven't tested this code.
```javascript
function toUTC(date) {
  return dateFns.subMinutes(date, date.getTimezoneOffset())
}
```
![image](https://user-images.githubusercontent.com/5661040/122326947-4ca63f00-cf57-11eb-9e53-3ddaa90853de.png)

**Reply by Shyam-Chen** [reply]
```ts
import * as d from 'date-fns';

const t = new Date();

const a = d.format(toUTC(t), "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
const b = t.toISOString();

function toUTC(date: Date) {
  return d.addMinutes(date, date.getTimezoneOffset());
}

console.log(a === b); // true
```
